qa/sc_evm_multiple_cert_submitters.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3
```

Comment:
I think we can simplify a lof the test. We just need to have 2 SC nodes and generate 1 certificate. After that we can check the nodes logs and verify that we have the expected error messages.

Reply:
Done